src/bin/vip-wp.js

```diff
@@ -289,9 +299,14 @@ commandWrapper( {
 			} );
 		} );
 
-		subShellRl.on( 'SIGINT', () => {
-			subShellRl.close();
-			process.exit();
+		subShellRl.on( 'SIGINT', async () => {
+			await trackEvent( 'wpcli_cancel_command', {
+				command: commandForAnalytics,
+			} );
+
+			//write out CTRL-C/SIGINT
+			subShellRl.write( '\x03\n' );
```

Comment:
Please just send the '\x03' character, without the newline, and you will be in business :) The runner is now expecting a data packet of length 1 with just the value of 0x3.